fiftyone/utils/sam2.py

```
@@ -111,6 +111,44 @@ def point_transform(self, points, img_hw, point_labels=None):
         )
 
+    def valid_image(self, curr_id):
+        if self.processor._is_image_set:
+            return curr_id == self.image_id
+        return False
+
+    @property
+    def original_size(self):
+        return self.processor._orig_hw[0]
+
+    def predict(
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `predict`

(ANN202)